### PR TITLE
HDDS-4393. Addressing test failures after master merge.

### PR DESCRIPTION
**Pull request by nandakumar131**
## What changes were proposed in this pull request?

Addressing test failures after master merge.

## What is the link to the Apache JIRA

HDDS-4393

### CODE REVIEW

File: hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/admin/scm/ScmAdmin.java [hunk 1]
```diff
@@ -20,6 +20,8 @@
 import org.apache.hadoop.hdds.cli.GenericCli;
 import org.apache.hadoop.hdds.cli.HddsVersionProvider;
 import org.apache.hadoop.hdds.cli.OzoneAdmin;
+import org.apache.hadoop.hdds.cli.SubcommandWithParent;
+import org.kohsuke.MetaInfServices;
```
Comment (mukul1987) [inline hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/admin/scm/ScmAdmin.java:24]: This is a new dependency, Can you please add detail about this ?

**Review by mukul1987 — CHANGES_REQUESTED** [review]

File: hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/admin/scm/ScmAdmin.java (hunk 1 above)
Comment (nandakumar131) [reply hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/admin/scm/ScmAdmin.java:24]: It's not a new dependency, it was added as part of HDDS-4046.

**Review by xiaoyuyao — APPROVED** [review]
LGTM, +1

**Review by mukul1987 — APPROVED** [review]
+1

**Pull request merged**